ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -92,13 +92,18 @@ public func toRACSignal<T: AnyObject, E>(producer: SignalProducer<T, E>) -> RACS
 /// Any `Interrupted` events will be silently discarded.
 public func toRACSignal<T: AnyObject, E>(producer: SignalProducer<T?, E>) -> RACSignal {
 	return RACSignal.createSignal { subscriber in
-		let selfDisposable = producer.start(next: { value in
-			subscriber.sendNext(value)
-		}, error: { error in
-			subscriber.sendError(error as NSError)
-		}, completed: {
-			subscriber.sendCompleted()
-		})
+		let selfDisposable = producer.start { event in
+			switch event {
+			case .Next(let value):
+				subscriber.sendNext(value)
+			case .Error(let error):
+				subscriber.sendError(error as NSError)
+			case .Completed:
+				subscriber.sendCompleted()
+			default:
```

Comment:
I like this PR for this line right here: users will be aware of their need to explicitly ignore events, which wasn't the case before due to the default arguments!